### PR TITLE
update dev dependencies, adjust tests

### PR DESCRIPTION
**Pull request by cgcostume**

### CODE REVIEW

File: source/auxiliaries.ts [hunk 1]
```diff
@@ -275,7 +275,7 @@ namespace auxiliaries {
      * Queries window.location.search.
      */
     export function GETsearch(): string {
-        return document.location.search;
+        return document.location!.search;
```
Comment (bwasty) [inline source/auxiliaries.ts]: With `window.location` you don't need an exclamation mark ([should be the same](https://stackoverflow.com/a/2430948/2858790)).
Same below.

File: source/webgl-operate.slim.ts [hunk 2]
```diff
@@ -51,8 +51,24 @@ export { KernelF32, KernelI32, KernelI8, KernelUI32, KernelUI8 } from './kernel'
...
+export { FontFace } from './text/fontface';
+export { FontLoader } from './text/fontloader';
+export { GlyphVertex, GlyphVertices } from './text/glyphvertices';
+export { Label } from './text/label';
+export { LabelGeometry } from './text/labelgeometry';
+export { Position2DLabel } from './text/position2dlabel';
+export { Position3DLabel } from './text/position3dlabel';
+export { Text } from './text/text';
+export { Typesetter } from './text/typesetter';
 export { LabelRenderPass } from './text/labelrenderpass';
```
Comment (bwasty) [inline source/webgl-operate.slim.ts]: Possible alternative: add a `./text/index.ts` with all these exports (same later for core, gltf, xr etc.)

File: test/object.test.ts [hunk 3]
```diff
@@ -39,7 +38,7 @@ class ObjectMock extends AbstractObject<string> {
     create(): string {
         this._object = this._fakeFail ? undefined : 'object';
         this._valid = !this._fakeFail;
-        return this._object;
+        return this._object ? this._object : 'undefined';
```
Comment (bwasty) [inline test/object.test.ts]: minor: `|| 'undefined'`?

File: source/webgl-operate.slim.ts (hunk 2 above)
Comment (cgcostume) [reply source/webgl-operate.slim.ts]: yes - we should do this

**Pull request merged**